pkg/kube/krt/processor.go

```diff
@@ -0,0 +1,251 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package krt
+
+import (
+	"sync"
+	"sync/atomic"
```

Comment:
why sync/atomic here rather than Uber's atomic (which seems to be the more commonly used for Istio)?

Reply:
This code is forked from k8s client-go

Reply:
makes sense, I figured it might be worth asking even though that was the likely answer